index.html

```diff
@@ -0,0 +1,122 @@
+                    <li>Remove mushrooms from the pot, squeeze them, and cut into smaller pieces (if they were big).</li><br>
+                    <li>Filter the water (where you soaked mushrooms) through several layers of <span><a htef="">cooking gauze</a></span>  to get rid of debris and sand. Make sure not to pour it away (!!!) since this is the most flavorful soup base! Into the same pot with filtered mushroom infusion put mushrooms, add more water (to have around 3 liters of liquid in total), salt, and black peppercorns. Place on the stove and bring it to a boil.</li><br>
+                    <li>As soon as it boils, there may be a foam. Remove it carefully with a skimmer, reduce heat and simmer for about 30-35 minutes. The readiness of mushrooms is determined when during cooking they begin to sink to the bottom of the pot.</li><br>
+                    <li>Meanwhile, peel and chop a potato, dice an onion, and shred a carrot.</li><br>
+                    <li>Heat a frying pan over medium heat and add 4 Tbsp of olive oil. Add diced onion and saute for 2-3 minutes stirring often and not letting it burn. Then add carrots and saute all together for about 5 more minutes. If you see that carrots are too dry, add a bit more oil.</li><br>
+                    <li>When mushrooms cook for 30-35 minutes, add chopped potatoes and simmer for additional 15-20 minutes. Once potatoes reach desired softness, add sauteed carrots with onions and continue to simmer the soup under a closed lid for another 10 minutes. At the same time when you add carrots, add the bay leaf but remove it from the soup in 15 minutes. If necessary, add more salt.
+                    </li><br>
+                    <li>Once your porcini mushroom soup is ready, turn the heat off but leave it under a lid to infuse for at least 2 minutes. But the longer it infuses, the more flavorful it will become. You’ll see that on the following day the mushroom taste will be much stronger.</li><br>
+                    <li>When serving, sprinkle with finely cut dill and add a bit of sour cream (if you wish).</li><br>
+                </ol>
+                <div class="instruction-img">
+                    <img src="/Users/Mac/Documents/CYF/RecipeCoursework/images/Ukrainian-porcini-mushroom-soup-1-1-scaled.jpeg" alt="image of chopped onion"/>
+                    <img src="/Users/Mac/Documents/CYF/RecipeCoursework/images/Ukrainian-Mushroom-Soup-4.jpeg" alt="image of fried onion"/>
+                    <img src="/Users/Mac/Documents/CYF/RecipeCoursework/images/Ukrainian-Mushroom-Soup-2.jpeg" alt="image of fried onion with carrot"/>
+                    <img src="/Users/Mac/Documents/CYF/RecipeCoursework/images/Ukrainian-Mushroom-Soup-3.jpeg" alt="mix and boil all ingridients together in a pot"/>
+                    <img src="/Users/Mac/Documents/CYF/RecipeCoursework/images/image.jpeg" alr="image of ready soup"/>
```

Comment:
`/Users/Mac/Documents/CYF/RecipeCoursework/images/Ukrainian-Mushroom-Soup-3.jpeg` is called using an absolute path. Basically the images will disappear when you move the folder around. 

Try `./images/Ukrainian-Mushroom-Soup-3.jpeg` instead. This is a relative path

Reply:
@JarrodFolinoFC  Thank you ! will try)